Recipe creation page

bfrieda opened the pull request:
related to issue #43 

<img width="1440" alt="image" src="https://github.com/algorizan/PrepPal/assets/98629526/e4c4a02a-9203-4550-8eb5-011de6739828">

<img width="1440" alt="image" src="https://github.com/algorizan/PrepPal/assets/98629526/3f7f5431-79c2-4675-af89-2e2dd9313af6">

lik5 commented on preppal-fe/src/pages/new-recipe.tsx:
```diff
@@ -0,0 +1,104 @@
+    return (
+        <Container className="d-flex justify-content-center">
+            <div className="w-100" style={{ maxWidth: '400px' }}>
+                <h2 className="text-center mb-4">Create Recipe</h2>
```
need to create more space between the nav bar and the top of the form

lik5 commented on preppal-fe/src/pages/new-recipe.tsx:
```diff
@@ -0,0 +1,104 @@
+                        </Stack>
+                    </Form.Group>
+
+                    <Form.Group controlId="recipeNutritionFacts" style={{ paddingBottom: '24px' }}>
```
Let's remove this

lik5 reviewed (commented):
We also need to make every input field larger, it looks kind of small.

bfrieda commented:
> We also need to make every input field larger, it looks kind of small.

The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out
But I can change it if you still think it's too small

hwbit commented on preppal-fe/src/pages/new-recipe.tsx:
```diff
@@ -0,0 +1,96 @@
+    );
+};
+
+export default LoginPage;
```
Change the export default to NewRecipe.
Also add new routing to app.ts
I think we just add a link for this page to the navbar and do cleanup later.

lik5 commented:
> > We also need to make every input field larger, it looks kind of small.
> 
> The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out But I can change it if you still think it's too small

Can you make it so that the form is about this size 
<img width="1440" alt="Screenshot 2024-02-12 at 1 34 54 PM" src="https://github.com/algorizan/PrepPal/assets/143126314/ed5374c4-0029-46b8-976a-64782033791a">

bfrieda commented:
> > > We also need to make every input field larger, it looks kind of small.
> > 
> > 
> > The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out But I can change it if you still think it's too small
> 
> Can you make it so that the form is about this size. Approximately like 2/3 of the screen width <img alt="Screenshot 2024-02-12 at 1 34 54 PM" width="1440" src="https://private-user-images.githubusercontent.com/143126314/304203305-ed5374c4-0029-46b8-976a-64782033791a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3Njc2NDcsIm5iZiI6MTcwNzc2NzM0NywicGF0aCI6Ii8xNDMxMjYzMTQvMzA0MjAzMzA1LWVkNTM3NGM0LTAwMjktNDZiOC05NzZhLTY0NzgyMDMzNzkxYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMlQxOTQ5MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYTZlNTI2ZjQ2NTg1OTM0NTI2OWY4MmM3MTU5N2NhYTY5NmY4OWE2YTRkMmIwMGNlODk3NWFkY2RmOTIyNzYwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.HOUg4--x4I9ilm7MWrQoROaePgOsHTHtv4zvUjAy3Ic">



> > > We also need to make every input field larger, it looks kind of small.
> > 
> > 
> > The screenshot was zoomed out to include everything, I updated the comment with how it looks now and not zoomed out But I can change it if you still think it's too small
> 
> Can you make it so that the form is about this size. Approximately like 2/3 of the screen width <img alt="Screenshot 2024-02-12 at 1 34 54 PM" width="1440" src="https://private-user-images.githubusercontent.com/143126314/304203305-ed5374c4-0029-46b8-976a-64782033791a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3Njc2NDcsIm5iZiI6MTcwNzc2NzM0NywicGF0aCI6Ii8xNDMxMjYzMTQvMzA0MjAzMzA1LWVkNTM3NGM0LTAwMjktNDZiOC05NzZhLTY0NzgyMDMzNzkxYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxMlQxOTQ5MDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYTZlNTI2ZjQ2NTg1OTM0NTI2OWY4MmM3MTU5N2NhYTY5NmY4OWE2YTRkMmIwMGNlODk3NWFkY2RmOTIyNzYwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.HOUg4--x4I9ilm7MWrQoROaePgOsHTHtv4zvUjAy3Ic">

I created a new issue #65 for this, since it's not a priority at the moment
